docs/en/tutorials/getting-started.rst

```diff
@@ -644,24 +639,22 @@ able to work with Doctrine 2. These assumptions are not unique to
 Doctrine 2 but are best practices in handling database relations
 and Object-Relational Mapping.
 
-
```

Comment:
Here I moved the first two bullets to the end of the list, so that we define the terms "owning side" and "inverse side" before using them. I also reorganized and reworded the other bullets, so that they flow from a discussion of one-to-one, many-to-one, then finally many-to-many.